integrations/QuoraPixel/browser.js

```diff
@@ -49,9 +49,12 @@ class QuoraPixel {
     const { event } = rudderElement.message;
     const eventsMapping = getHashFromArrayWithDuplicate(this.eventsToQPEvents);
     const trimmedEvent = event.toLowerCase().trim();
-    const events = eventsMapping[trimmedEvent];
+    const events = eventsMapping[trimmedEvent] || [];
 
-    if (!events || events.length === 0) {
+    if (events.length === 0) {
+      logger.debug(
```

Comment:
Why not warn?

Reply:
Yes, we can use `warn` in this case. Updating it to `warn`.